### PR TITLE
Feature/cheatsheet snippets

### PR DESCRIPTION
**Pull request by Estela-Ex**
@hhkaos

### CODE REVIEW

File: cheatsheet/main.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    const span = document.createElement("span");
+    span.innerText = body[i];
+    outputElementBody.appendChild(span);
+  }
+};
+const onChangeGeneralSelector = (selectedValue) => {
+  const selectedData = file[selectedValue];
+
+  outputElementPrefix.textContent = `${selectedData.prefix}`;
+  // outputElementBody.textContent = `${selectedData.body}`;
```
Comment (hhkaos) [inline cheatsheet/main.js:71]:
```suggestion
```

File: cheatsheet/main.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  createBody(selectedData.body);
+  outputElementDescription.textContent = `${selectedData.description}`;
+};
+const onChangelanguageSelect = (e) => {
+  while (generalSelector.firstChild) {
```
Comment (hhkaos) [inline cheatsheet/main.js:76]: Same as previously: 

`generalSelector.innerHTML = '';`

**Review by hhkaos — CHANGES_REQUESTED** [review]
Good @Estela-Ex! I just made some comments and suggestions to your code.

**Comment by hhkaos** [comment]
I also noticed if you don't add `text-wrap: wrap;`to `#output-body` in the CSS some script won't show properly. Example:
Screenshot 2023-10-18 at 13 09 49

**Comment by hhkaos** [comment]
@Estela-Ex with the last changes to master branch, this application breaks. I think the reason is:
* We have removed the `scss.json` file
* And some snippets `prefix` now contains an array instead of a string (e.g. [\["featureReductionClusterProps", "clusterConfig"\]](https://github.com/Esri/arcgis-js-vscode-snippets/blob/master/snippets/javascript.json#L226)). In these cases we should display the first element of the array. 

If you need any additional help, let me know.
Thanks!

**Comment by Estela-Ex** [comment]
@hhkaos I did the request changes

**Review by hhkaos — APPROVED** [review]
LGTM

**Comment by hhkaos** [comment]
Thank you very much for your contributions @Estela-Ex! This, I believe, will greatly help us in improving the documentation and use of this extension. I'm sure the entire user community appreciates the effort you've done! ❤️

**Pull request merged**